### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Disability Benefits Crew - Core Form Team, 21-526ez, Side Navigation

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#126217]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Consider:** In the process list version, the unavailable steps have the disabled treatment due to the steps being non-navigable. While disabled elements are exempt from color contrast minimums, we generally think of disabled as disabled fields, disabled options in a select/combobox, disabled buttons. However, this component is meant to be a wayfinding tool and the wayfinding context (seeing the total number of steps, and what each step is) may be lost for some users due to the low contrast of non-selectable steps. Users with low vision, or high contrast settings, may not be able to perceive those steps. So although those links are non-navigable, consider meeting contrast minimums in your color choice, so that all users can perceive and understand the full context of the element and receive all of the wayfinding benefits. 

<img width="500" alt="contrast" src="https://github.com/user-attachments/assets/73b01447-a9db-4f2d-85ec-3453cae81798" />

<hr/>

- [ ] **Consider:**. The sidenav component, on small screens, becomes an [accordion interaction](https://design.va.gov/storybook/?path=/story/components-va-sidenav--default). While this component is accessible, there is the potential for a reduction in wayfinding due to the static accordion text "Select a step" - this "step" language is not reflected in the heading below, only the step value.  As there was much discussion around this, my only point is to consider the intent of wayfinding and how that is reflected on screen, so that users understand where they are and what's next. 

<hr/>

- [ ] **Consider:** Your accordion visual does not match what is in VADS, and further, it is styled more like a select or combobox component. That styling may confuse users, who are expecting specific behaviors based on the UI. Consider how the accordion styling may impact user expectation. @humancompanion-usds 

### side nav accordion
<img width="538" height="238" alt="side nav accordion" src="https://github.com/user-attachments/assets/7da3febb-cd06-4b8c-91b4-399ff576011b" />

### Your accordion treatment
<img width="362" height="227" alt="confusing UI" src="https://github.com/user-attachments/assets/bd74fc7a-7ee4-4177-b252-c9b8ed180073" />

### Other components - selects, comboboxes, etc.
VADS select
<img width="499" height="100" alt="VADS select" src="https://github.com/user-attachments/assets/140024a9-2108-441e-a26a-140b044bf18d" />

USWDS combobox
<img width="512" height="102" alt="USWDS combobox" src="https://github.com/user-attachments/assets/e9fab4af-205a-44f9-b207-aba9e3eca6a6" />

Lightning design system combobox
<img width="1920" height="600" alt="LDS combobox" src="https://github.com/user-attachments/assets/d255db7b-cd02-498c-9852-670ab320da36" />

<hr/>

- **Note:** During the Midpoint review, I brought up the concept of this component being a combobox (or even an action menu). VADS doesn't have an action menu component, and the VADS implementation of a combobox doesn't incorporate this use case - a [select only combobox](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/). The VADS combobox includes typeahead and is used more for form entry. Due to this, I have pulled back that recommendation. 

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app

**Comment by bianca-rivera** [comment]
Closing since we have explored and incorporated feedback from this ticket
